### ISSUE TITLE
Missing Relu in DenseNet

### ISSUE THREAD

**Issue by daavoo** [keras-team/keras-applications#31]
In the official DenseNet implementations ([caffe](https://github.com/liuzhuang13/DenseNetCaffe/blob/master/make_densenet.py#L57), [torch](https://github.com/liuzhuang13/DenseNet/blob/master/models/densenet.lua#L43), [pytorch](https://github.com/gpleiss/efficient_densenet_pytorch/blob/master/models/densenet.py#L153)) and others ([mxnet](https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/gluon/model_zoo/vision/densenet.py#L105), [torchvision](https://github.com/pytorch/vision/blob/master/torchvision/models/densenet.py#L222)) a Relu activation is applied after the [last batch norm](https://github.com/keras-team/keras-applications/blob/master/keras_applications/densenet.py#L221) and before the global pooling, however this activation does not exist in this keras implementation.

**Comment by taehoonlee** [comment]
You are right. Thank you for the report, @daavoo. Please see [the commit](https://github.com/keras-team/keras-applications/commit/e8a7474111e5b77d340d2c4204130b83e9ed69b8). The top-5 accuracies have been improved by about 0.3% because of your help :)

**Comment by daavoo** [comment]
I'm glad to hear that! Closing this
